integreat_cms/core/settings.py

```diff
@@ -64,6 +64,26 @@
 
 TEST_REGION_SLUG: Final[str] = "testumgebung"
 
+#: Reserved region slugs, that are used in the webapp
+RESERVED_REGION_SLUGS = [
```

Comment:
```suggestion
RESERVED_REGION_SLUGS: Final[list[str]] = [
```
Technically, this is not really required, but I think it's helpful to prevent the settings from being used incorrectly or being overridden...